test/metabase/util/embed_test.clj

```diff
@@ -1,8 +1,9 @@
 (ns ^:mb/once metabase.util.embed-test
   (:require
    [buddy.sign.jwt :as jwt]
-   [clojure.test :refer :all]
+   [clojure.test :refer [deftest is testing]]
```

Comment:
Just changed this to be a little more specific in what we're importing

Reply:
We actually do usually use `[clojure.test :refer :all]` in test namespaces. Not a big deal either way though.